### PR TITLE
Options definitions split

### PR DESCRIPTION
**Pull request by AndreasEhret**
I've mainly splitted OptionsDefinitions by extracting the options print part to separate class so that we can move OptionsDefinitions to common module. I've used a new namespace as proposed: net.adoptopenjdk.icedteaweb. This PR is based on #125

### CODE REVIEW

File: core/src/main/java/net/sourceforge/jnlp/runtime/HtmlBoot.java [hunk 1]
```diff
@@ -47,9 +47,8 @@
 import net.sourceforge.jnlp.util.optionparser.OptionParser;
 import net.sourceforge.swing.SwingUtils;
 
-import javax.swing.JFrame;
-import java.awt.Point;
-import java.awt.Rectangle;
+import javax.swing.*;
+import java.awt.*;
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/runtime/HtmlBoot.java]: please do not use * imports

File: core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                break;
+            case TextsProvider.POLICY_EDITOR:
+                printOptions(getPolicyEditorOptions());
+                break;
+            default:
+                break;
+        }
+    }
+
+    private static void printOptions(final List<OPTIONS> options) {
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java]: null check for options

File: core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        final List<OPTIONS> l = new ArrayList<>();
+        l.addAll(getJavaWsRuntimeOptions());
+        l.addAll(getJavaWsControlOptions());
+        //trustall is not returned by getJavaWsRuntimeOptions
+        //or getJavaWsControlOptions, as it is not desired in documentation
+        l.add(OPTIONS.TRUSTALL);
+        return l;
+    }
+
+    public static void main(String[] args) throws IOException {
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java]: check that args is not null and has size > 0

File: core/src/main/java/net/sourceforge/jnlp/runtime/HtmlBoot.java (hunk 1 above)
Comment (AndreasEhret) [reply core/src/main/java/net/sourceforge/jnlp/runtime/HtmlBoot.java]: done

File: core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java [hunk 4]
```diff
@@ -124,6 +123,11 @@
     }
 
     public static void main(String[] args) throws IOException {
+        if ((args == null) || (args.length == 0)) {
+            System.out.println(String.format("Missing one of the arguments: %s | %s | %s",
```
Comment (hendrikebbers) [inline core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java]: LOG.error ;)

File: core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java (hunk 2 above)
Comment (hendrikebbers) [reply core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java]: now you changed the functionality of the method.

I would just add Objects.requireNotNull(options); as first line of the method. if options is null a NPE would be thrown in the old code somewhere in the logic. By adding this line NPEs are much easier to find.

Comment (AndreasEhret) [reply core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java]: done

File: core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java (hunk 3 above)
Comment (AndreasEhret) [reply core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java]: done

File: core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java (hunk 4 above)
Comment (AndreasEhret) [reply core/src/main/java/net/sourceforge/jnlp/util/OptionsDefinitionsPrinter.java]: done

**Pull request merged**